WooCommerce/Classes/ServiceLocator/PushNotification.swift

```diff
@@ -4,6 +4,11 @@ import Foundation
 import struct Yosemite.Note
 #elseif canImport(NetworkingWatchOS)
 import struct NetworkingWatchOS.Note
+#elseif canImport(Networking)
+import struct Networking.Note
+#endif
```

Comment:
Can you explain a little bit why is this import needed?

If I remember correctly, `Yosemite.Note` is the same as `Networking.Note`

Is it to be able to link in in the extension? If so maybe we should change the initial `Yosemite.Note` to not confuse any other reader 😅

Reply:
Yosemite.Note:
`public typealias Note = Networking.Note`

I think I added it because Yosemite is not available for notification extension?
If you look at `OrderNotificationDataService.swift` it imports `Networking`

Reply:
I think it is safe to change the initial `import Yosemite.Note` to `import Networking.Note`